### ISSUE TITLE
client.title_stats limit updated to 500 from 1000. So calling title stats with limit > 500 fails. 

### ISSUE THREAD

**Issue by biggydbs** [isFakeAccount/psnawp#39]
Also If it is called without limit parameter, by default the limit gets set to 1000 so failing as well.

Attaching the response:
```
{
  "error": {
    "reason": "BadRequest",
    "code": 3239940,
    "source": "Gamelist",
    "message": "invalid request, requested limit=1000 allowed limit=500",
    "referenceId": "30d04dab-6208-1031-a813-34d1df1bb537"
  }
}
```

**Comment by Dev-Nath** [comment]
I am also having this issue. It was working about 2-3 days back. Is there was way to go back to a previous version maybe? @isFakeAccount

**Comment by isFakeAccount** [comment]
Looks like they changed the way the endpoint behaves, I can fix it as soon as I get some time.

**Comment by isFakeAccount** [comment]
> I am also having this issue. It was working about 2-3 days back. Is there was way to go back to a previous version maybe? @isFakeAccount

@Dev-Nath Going back to previous version won't help, It has been 1000 since the beginning. The current version did not change anything in title_stats.py

**Comment by isFakeAccount** [comment]
The issue is now fixed in version 1.2.3 https://github.com/isFakeAccount/psnawp/releases/tag/v1.2.3
